lib/msf/core/post/linux/packages.rb

```diff
@@ -0,0 +1,74 @@
+# -*- coding: binary -*-
+
+module Msf
+  class Post
+    module Linux
+      module Packages
+        include ::Msf::Post::Linux::System
+
+        #
+        # Determines the version of an installed package
+        #
+        # @param package The package name to check for
+        # @return [Rex::Version] nil if OS is not supported or package is not installed
+        #
+        def installed_package_version(package)
+          info = get_sysinfo
+
+          if ['debian', 'ubuntu'].include?info[:distro]
+            package_version = cmd_exec("dpkg -l #{package} | grep \'^ii\'")
+            return nil unless package_version.start_with?('ii')
+
+            package_version = package_version.split(' ')[2]
+            package_version = package_version.gsub('+', '.')
+            return Rex::Version.new(package_version)
+          elsif ['redhat', 'fedora'].include?(info[:distro])
+            package_version = cmd_exec("rpm -q #{package}")
+            return nil if package_version.include?('is not installed')
```

Comment:
I haven't tested it, but I'm wondering if `rpm` returns a message in English on a non-English installation.

This also applies to the system commands called in this method.

Reply:
Good thinking, I'm not sure